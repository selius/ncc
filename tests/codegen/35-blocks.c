int main()
{
	int a;
	a = 2;
	{

	}

	{
		{
		}
	}

	{
		__print_int(111);
	}

	{
		int b;
		b = 4;
		__print_int(a);
		__print_int(b);
		{
			int c;
			c = 7;
			a = 6;
			__print_int(c);
			__print_int(b);
			__print_int(a);
			return 0;
		}
		a = b;
	}

	return a;
}
