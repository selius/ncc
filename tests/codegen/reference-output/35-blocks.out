111
2
4
7
4
6
